pod-batch.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+# this value should be differernt when running script again, as each time a script is run a new Pod is created
+  name: pod1
+spec:
+  containers:
+  - name: barepod
+  # Docker image name
+    image: 172.30.223.233:5000/sample-project/batch-cgs
+    command: ["python", "groupsforsections.py"]
+    env:
+    # the below 2 are environmental variables in Open Shift
```

Comment:
You're right that you can't find the valid values aren't in the right place.  But it is possible to give an error message and refuse to continue if good values aren't provided.  If there are good default values it is great to provide them.  If appropriate default values are hard to specify it is probably better to throw a fit and refuse to run until good values are provided.